docs/configuration-reference/platforms/baremetal.md

```diff
@@ -149,7 +151,8 @@ os_version = var.custom_default_os_version
 | `ssh_pubkeys`               | List of SSH public keys for user `core`. Each element must be specified in a valid OpenSSH public key format, as defined in RFC 4253 Section 6.6, e.g. "ssh-rsa AAAAB3N...". |        -         | list(string) |   true   |
 | `os_version`                | Flatcar Container Linux version to install. Version such as "2303.3.1" or "current".                                                                                         |    "current"     |    string    |  false   |
 | `os_channel`                | Flatcar Container Linux channel to install from ("flatcar-stable", "flatcar-beta", "flatcar-alpha", "flatcar-edge").                                                         | "flatcar-stable" |    string    |  false   |
-| `enable_tls_bootstrap`      | Enable TLS bootstraping for Kubelet.                                                                                                                                         |      true        |     bool     |  false   |
+| `enable_tls_bootstrap`      | Enable TLS bootstraping for Kubelet.                                                                                                                                         |       true       |     bool     |  false   |
+| `encrypt_pod_traffic`       | Enable in-cluster pod traffic.                                                                                                                                               |      false       |     bool     |  false   |
```

Comment:
We don't expose the MTU here, I created an issue for this: https://github.com/kinvolk/lokomotive/issues/976

Reply:
created PR for it #977